spec/wor/test_services/some_service.rb

```diff
@@ -71,6 +77,6 @@ def method_without_rescue
   protected
 
   def base_url
-    'https://baseurl.com'
+    'https://baseurl.com' unless base_url_empty
```

Comment:
```suggestion
    base_url_empty || 'https://baseurl.com' 
```

Reply:
if use this the method `base_url ` return a boolean, and i can't testing when is empty